### PR TITLE
Adding references to EFF's Public Project Code of Conduct.

### PR DESCRIPTION
**Pull request by jgillula**

### CODE REVIEW

**Comment by pipboy96** [comment]
Add `CODE_OF_CONDUCT.md` file in the root so GitHub will remove the "no code of conduct" warning.

**Comment by pipboy96** [comment]
If this change is done incorrectly, undo it using `git reset --hard  5f2f5bf && git push --force`.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ If you get stuck we have two publicly-archived mailing lists: the https-everywhe
 
 You can also find more information on about HTTPS Everywhere on our [FAQ](https://www.eff.org/https-everywhere/faq) page.
 
+Also, please remember that this Project is Governed by [EFF's Public Projects Code of Conduct](https://www.eff.org/pages/eppcode).
```
Comment (pipboy96) [inline CONTRIBUTING.md]: Is capital `Governed` here on purpose (e.g. it has a special meaning legally)?

Comment (jgillula) [reply CONTRIBUTING.md]: Nope, fixed!

**Pull request merged**